locales/zh_CN/LC_MESSAGES/hello.po

```diff
@@ -213,125 +210,117 @@ msgstr "（可用: 该系统上有最完整的支持；不可用: 由于平台
+"into a language that is **compiled, statically-typed, lexically-scoped, parallel and differentiable**."
 msgstr ""
-"Taichi的计算是在 **内核** "
-"中进行的。内核的参数必须显式指定类型。Taichi内核与函数中所用的语法，看起来和Python的很像，Taichi的前端编译器会将其转换为 "
-"**编译型，静态类型，有词法作用域，并行执行，可微分** 的语言。"
+"计算发生在 Taichi 的 **内核(kernel)** 模块中。内核的参数必须显式指定类型。Taichi 内核与函数中所用的语"
+"法，看起来和 Python 的很像，然而 Taichi 的前端编译器会将其转换为 **编译型，静态类型，有词法作用域，并"
+"行执行，可微分** 的语言。"
 
 #: ../../hello.rst:120
 msgid ""
-"Taichi **functions**, which can be called by Taichi kernels and other "
-"Taichi functions, should be defined with the keyword ``ti.func``."
+"Taichi **functions**, which can be called by Taichi kernels and other Taichi functions, should be "
+"defined with the keyword ``ti.func``."
 msgstr ""
+"Taichi 的 **函数** 可以被 Taichi 内核和其他 Taichi 函数调用，你应该使用关键字 ``ti.func`` 来进行定义。"
 
 #: ../../hello.rst:124
-#, fuzzy
 msgid ""
-"**Taichi-scopes v.s. Python-scopes**: everything decorated with "
-"``ti.kernel`` and ``ti.func`` is in Taichi-scope, which will be compiled "
-"by the Taichi compiler. Everything else is in Python-scopes. They are "
-"simply Python code."
+"**Taichi-scopes v.s. Python-scopes**: everything decorated with ``ti.kernel`` and ``ti.func`` is in "
+"Taichi-scope, which will be compiled by the Taichi compiler. Everything else is in Python-scopes. "
+"They are simply Python code."
 msgstr ""
-"**Taichi作用域 与 Python作用域**：任何带有 ``@ti.kernel`` 和 ``@ti.func`` "
-"装饰器的函数体都在Taichi作用域中，这些代码会被Taichi编译。而在Taichi作用域之外的就是Python作用域了，它们是单纯的Python代码。"
+"**Taichi 作用域与 Python 作用域**：任何被 ``@ti.kernel`` 和 ``@ti.func`` 修饰的函数体都处于 Taichi 作"
+"用域中，这些代码会由 Taichi 编译器编译。而在 Taichi 作用域之外的就都是 Python 作用域了，它们是单纯的 "
+"Python 代码。"
 
 #: ../../hello.rst:129
 msgid ""
-"Taichi kernels must be called in the Python-scope. I.e., **nested kernels"
-" are not supported**. Nested functions are allowed. **Recursive functions"
-" are not supported for now**."
-msgstr "Taichi内核只有在Python作用域中才能调用，也就是说，**我们不支持嵌套内核**。嵌套函数支持，但 **递归函数不支持** 。"
+"Taichi kernels must be called in the Python-scope. I.e., **nested kernels are not supported**. Nested "
+"functions are allowed. **Recursive functions are not supported for now**."
+msgstr ""
+"Taichi 内核只有在 Python 作用域中才能调用，也就是说，**我们不支持嵌套内核**。同时有嵌套函数支持，但 **"
+"递归函数不支持** 。"
 
 #: ../../hello.rst:132
 msgid "Taichi functions can only be called in Taichi-scope."
-msgstr "Taichi函数只有在Taichi作用域中才能调用。"
+msgstr "Taichi 函数只有在 Taichi 作用域中才能调用。"
 
 #: ../../hello.rst:134
-#, fuzzy
 msgid ""
-"For those who come from the world of CUDA, ``ti.func`` corresponds to "
-"``__device__`` while ``ti.kernel`` corresponds to ``__global__``."
+"For those who come from the world of CUDA, ``ti.func`` corresponds to ``__device__`` while ``ti."
+"kernel`` corresponds to ``__global__``."
 msgstr ""
-"用CUDA世界的话说，就是 ``ti.func`` 对应于 ``__device__``，``ti.kernel`` 对应于 "
-"``__global__``。"
+"用 CUDA “世界”的话来说，相当于 ``ti.func`` 对应 ``__device__``，``ti.kernel`` 对应 ``__global__``。"
```

Comment:
IMHO the en original text isn't so percise&calm here.. do you have any walk around suggestions?

Reply:
How about this?
> 如果用 CUDA 来做类比的话， ``ti.func`` 就像是 ``__device__``，``ti.kernel``  就像是 ``__global__``。

Reply:
Sounds great!